src/themes/fonts.js

```diff
@@ -0,0 +1,70 @@
+const font_opensans = new FontFace("OpenSans", `url(https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap)`);
+
+export const opensans = {
+  type: {
+    primary: font_opensans,
+  },
+  weight: {
+    regular: "400",
+    bold: "700",
+    extrabold: "800",
+    black: "900",
+  },
+  size: {
+    s1: 12,
+    s2: 14,
+    s3: 16,
+    m1: 20,
+    m2: 24,
+    m3: 28,
+    l1: 32,
+    l2: 40,
+    l3: 48,
+  },
+};
+
+export const rockwell = {
```

Comment:
Not sure if we should leave these in as a proof-of-concept or potentially even for backwards compatibility... I would say we can probably drop Rockwell and TW Century MT in the near future and only use the MC2050 fonts, but we'll have to hash this all out with Comms first. Please leave these in for now 👍